Assets/Scripts/Player/Body/SeveredBodyPart.cs

```diff
@@ -0,0 +1,14 @@
+using UnityEngine;
+
+namespace Player.Body
+{
+    /// <summary>
+    /// MonoBehaviour exists to ensure a severed body part behaves as expected.
+    /// Should be attached to any game object that will be spawned as a piece severed from a body
+    /// </summary>
+    [RequireComponent(typeof(Rigidbody), typeof(Item), typeof(Collider))]
+    public class SeveredBodyPart : MonoBehaviour
+    {
+        
+    }
```

Comment:
It's not explained why an empty monobehaviour is needed

Reply:
To force all severed limbs to have a Rigibody, Item and Collider components. So you can be sure heads roll around on the ground and can be picked up, etc.

Reply:
But having this script is not enforced. How do you feel about making some checks when you sever the limb, and throw a warning if something isn't right?

Reply:
Thats... actually a great idea :D Added check and warning to BodyPart.cs